File: src/main/java/instance/os/SnapshotRequest.java
package instance.os;

import org.jfree.chart.JFreeChart;
import se.sics.kompics.Event;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-08
 *
 */

public class SnapshotRequest extends Event {

	private JFreeChart chart;

	public SnapshotRequest() {
		
	}
	
	public void setChart(JFreeChart chart) {
		this.chart = chart;
	}

	public JFreeChart getCPULoadChart() {
		return chart;
	}
	
}

File: src/main/java/instance/os/SnapshotRequest.scala
package instance.os

import se.sics.kompics.Event
import reflect.BeanProperty
import org.jfree.chart.JFreeChart

/**
 * @author Amir Moulavi
 * @date 2011-07-16
 *
 */
class SnapshotRequest extends Event {

  @BeanProperty var cpuLoadChart:JFreeChart = _

}